astronomer/providers/amazon/aws/example_dags/delete_stale_emr_vpcs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
```

Comment:
Please give some context of what this script is intended to do at the beginning of the script